applications/luci-app-samba4/htdocs/luci-static/resources/view/samba4.js

```diff
@@ -39,13 +39,17 @@ return view.extend({
 		o = s.taboption('general', form.Value, 'description', _('Description'));
 		o.placeholder = 'Samba4 on OpenWrt';
 		
+		s.taboption('general', form.Flag, 'enable_extra_tuning', _('Enable extra Tuning'),
+			_('Enable some community driven tuning parameters, that may improve write speeds and better operation via WiFi.\
+			Not recommend if multiple clients write to the same files, at the same time!'));
+		
 		s.taboption('general', form.Flag, 'disable_async_io', _('Force synchronous  I/O'),
 			_('On lower-end devices may increase speeds, by forceing synchronous I/O instead of the default asynchronous.'));
 
-		o = s.taboption('general', form.Flag, 'macos', _('Enable macOS compatible shares'),
```

Comment:
Why did you changed that as well?
This does not match your commit message

Reply:
its just a minor change, the assignment is not needed, thats all